pkg/harvester/l10n/en-us.yaml

```diff
@@ -1607,6 +1608,7 @@ advancedSettings:
     'harv-kubeconfig-default-token-ttl-minutes': 'TTL (in minutes) applied on Harvester administration kubeconfig files. Default is 0, which means to never expire.'
     'harv-longhorn-v2-data-engine-enabled': 'Enable the Longhorn V2 data engine. Default is false. <ul><li>Changing this setting will restart RKE2 on all nodes. This will not affect running VM workloads.</li><li>If you see "not enough hugepages-2Mi capacity" errors when enabling this setting, wait a minute for the error to clear. If the error remains, reboot the affected node.</li></ul>'
     'harv-additional-guest-memory-overhead-ratio': 'The ratio for kubevirt to adjust the VM overhead memory. The value could be zero, empty value or floating number between 1.0 and 10.0, default to 1.5.'
+    'harv-upgrade-config': 'Configure image preloading and VM restore options for upgrades. <a href="https://docs.harvesterhci.io/v1.4/advanced/index#upgrade-config" target="_blank" rel="noopener">settings/upgrade-config</a>'
```

Comment:
[Nit] Ideally, the link should be fetched dynamically based on the Harvester version.
Ref: https://github.com/harvester/harvester-ui-extension/blob/main/pkg/harvester/utils/feature-flags.js#L4-L8